lib/screen/book_screen.dart

```diff
@@ -31,14 +36,103 @@ class _BookScreenState extends State<BookScreen> {
   }
 
   Widget _appBar(BuildContext context) => SliverAppBar(
-    floating: true,
-    snap: true,
-    title: Text("Home",
-        style: TextStyle(
-            fontWeight: FontWeight.bold,
-            fontSize: 25,
-            color: Theme.of(context).accentColor
-        )
+    floating: false,
+    snap: false,
+    toolbarHeight: 250,
+    title: Column(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        /*CircleAvatar(
+          backgroundColor: kNewAccentColor,
+          minRadius: 5.0,
+          child: CircleAvatar(
+            radius: 15.0,
+            backgroundImage: AssetImage('assets/images/avatar.png'),
+          ),
+        ),*/
+        Padding(
+          padding: EdgeInsets.only(left: 15, top: 10),
+          child: Text(
+            "Hi,",
+            style: TextStyle(
+                color: kHintTextColorLight
+            ),
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.only(left: 15),
+          child: Text(
+            user.displayName != null ? user.displayName.split(' ').first : "Welcome Back",
```

Comment:
Displayed the name from FirebaseAuth, in case the attribute is null, displayed welcome back. Also split the name because Firebase send the full name